src/challengePaths/2021/YouRobot.ts

```diff
@@ -0,0 +1,139 @@
+import { round, runChoice, Slot, Stat, visitUrl } from "kolmafia";
+import { have } from "../../lib.js";
+import { get } from "../../property.js";
+import { $path, $skill, $slot, $stat } from "../../template-string.js";
+
+export const path = $path`You, Robot`;
+
+type Upgrade = "robot_shirt" | "robot_energy" | "robot_potions";
+
+/**
+ * Runs one turn scavenging in the scrapheap
+ */
+export function scavenge(): void {
+  visitUrl("place.php?whichplace=scrapheap&action=sh_scrounge");
+}
+
+/**
+ * Spends one turn collecting energy in the scrapheap
+ */
+export function collectEnergy(): void {
+  visitUrl("place.php?whichplace=scrapheap&action=sh_getpower");
+}
+
+/**
+ * @returns the energy cost for the next use of the chronolith
+ */
+export function expectedEnergyNextCollect(): number {
+  const raw = (25 + get("youRobotPoints")) * 0.85 ** get("_energyCollected");
+  return round(raw);
+}
+
+/**
+ * Spends energy at the chronolith to get 10 adventures
+ */
+export function visitChronolith(): void {
+  visitUrl("place.php?whichplace=scrapheap&action=sh_chronobo");
+}
+
+/**
+ * @returns how much energy we expect the next chronolith use to take
+ */
+export function expectedChronolithCost(): number {
```

Comment:
It’s not clear from name or doc how this differs from expectedEnergyNextCollect

Reply:
This covers how much the chronolith will cost, while expectedEnergyNextCollect covers how much energy you'll get next time you collect energy.

What would you prefer they be called?

Reply:
Oh I think I misunderstood